services/blockchain-connector/shared/sdk/interoperability.js

```diff
@@ -46,6 +36,26 @@ const getChainAccount = async (chainID) => {
 	}
 };
 
+const getMainchainID = async () => {
+	try {
+		// const mainchainID = await invokeEndpoint('interoperability_getMainchainID', { chainID });
```

Comment:
If given method `interoperability_getMainchainID` requires `chainID` parameter as defined in L41, then we should add chainID as input parameter to `getMainchainID`, please update accordingly.
```suggestion
const getMainchainID = async (chainID) => {
	try {
		// const mainchainID = await invokeEndpoint('interoperability_getMainchainID', { chainID });
```

Reply:
@nagdahimanshu  The method signature is not yet defined but I expect no params to be passed here.

@priojeetpriyom Please cache the mainchainID here, as requested previously.

```suggestion
const getMainchainID = async () => {
	try {
		// const mainchainID = await invokeEndpoint('interoperability_getMainchainID');
```

Reply:
Applied.